PSKoans/Public/Show-Karma.ps1

```diff
@@ -80,6 +84,32 @@ function Show-Karma {
                 $KoanLocation | Invoke-Item
             }
         }
+        'OpenFile' {
+            try {
+                $Results = Get-Karma @GetParams
+            }
+            catch {
+                $PSCmdlet.ThrowTerminatingError($_)
+            }
+            $Editor = Get-PSKoanSetting -Name Editor
+            $FilePath = Get-PSKoan -Topic $Results.CurrentTopic.Name -Scope User | Select-Object -ExpandProperty Path
+            $LineNumber = $Results.CurrentTopic.FailedLineNumber
+            switch ($Editor) {
+                { $_ -in 'code', 'code-insiders' } {
+                    $FullArgument = '--goto "{0}":{1} --reuse-window' -f $FilePath, $LineNumber
+                    Start-Process $Editor -ArgumentList $FullArgument
+                }
+                atom {
+                    $FullArgument = '"{0}":{1}' -f $FilePath, $LineNumber
+                    Start-Process $Editor -ArgumentList $FullArgument
+                }
+                {$_ -in 'powershell_ise', 'isep' } {
+                    $FullArgument = "{0}" -f $FilePath
+                    Start-Process -FilePath $Editor -ArgumentList $FullArgument
+                }
+                Default {Invoke-Item $FilePath}
```

Comment:
🤔 We should probably have the default here still call $Editor if there's anything in that, so maybe something like this?

```suggestion
                default {
                    if ($Editor) {
                        $FullArgument = "{0}" -f $FilePath
                        & $Editor $FullArgument
                    }
                    else {
                        Invoke-Item $FilePath
                    }
                }
```